### ISSUE TITLE
a bug?

### ISSUE THREAD

**Issue by ling2433** [FengMoeTeam/NHentai-android#78]
when I search for "kiyokawa", the program will stop running.

**Comment by ling2433** [comment]
The program version is 1.8.3（35）

**Comment by ArchieMeng** [comment]
yes, a bug. Caused by [Kiyokawa Zaidan] Kaa-san de Suma Sechainasai [Chinese] [新手獨自一人漢化]
> https://e-hentai.org/g/708101/3ff52b2f56/

**Comment by NotCR** [comment]
Fixed in latest version with commit (https://github.com/FengMoeTeam/NHentai-android/commit/e1db9e798181bd28c58450c556754c41445b6242)

**Comment by NotCR** [comment]
@ling2433 can you please test Sith latest version

**Comment by wightwulf1944** [comment]
@NotCR tested with 1.8.4 beta 36
Issue fixed
